bvm/ballerina-core/src/main/java/org/ballerinalang/util/observability/ObservabilityUtils.java

```diff
@@ -195,7 +195,8 @@ public static void stopObservation(ObserverContext observerContext) {
      * before calling this method. Otherwise it can cause a {@link NullPointerException}.
      */
     public static ObserverContext getParentContext(Context context) {
```

Comment:
This comment is no longer valid:

@return the parent {@link ObserverContext}. Make sure to check {{@link #isObservabilityEnabled()}}
      * before calling this method. Otherwise it can cause a {@link NullPointerException}.

Please remove it

Reply:
Fixed in bee0252